kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -127,6 +127,15 @@
   private static final String AVRO_DATA_CACHE_SIZE_DOC =
       "The size of the cache to use when converting schemas from Avro to Kafka Connect";
 
+  public static final String CONVERT_DOUBLE_SPECIAL_VALUES_CONFIG =         "convertDoubleSpecialValues";
+  public static final ConfigDef.Type CONVERT_DOUBLE_SPECIAL_VALUES_TYPE =   ConfigDef.Type.BOOLEAN;
+  public static final Boolean CONVERT_DOUBLE_SPECIAL_VALUES_DEFAULT =       true;
+  public static final ConfigDef.Importance CONVERT_DOUBLE_SPECIAL_VALUES_IMPORTANCE =
+      ConfigDef.Importance.LOW;
+  public static final String CONVERT_DOUBLE_SPECIAL_VALUES_DOC =
+          "Should +/- Infinity and NaN be converted to Double.MAX_VALUE and"
```

Comment:
Should clarify that +Infinity is converted to MAX while -Infinity and NaN are converted to MIN